CHANGELOG.md

```diff
@@ -1,3 +1,270 @@
+  `pacote@2.7.38`:
+  * [zkat/pacote#102](https://github.com/zkat/pacote/pull/102) Fix issue with tar extraction and special characters.
+  * Enable loose semver parsing in some missing corner cases.
+  ([@colinrotherham](https://github.com/colinrotherham), [@zkat](https://github.com/zkat), [@mcibique](https://github.com/mcibique))
+* [`e2f815f87`](https://github.com/npm/npm/commit/e2f815f87676b7c50b896e939cee15a01aa976e4)
+  [#17104](https://github.com/npm/npm/pull/17104)
+  Write an empty str and wait for flush to exit to reduce issues with npm
+  exiting before all output is complete when it's a child process.
+  ([@zkat](https://github.com/zkat))
+* [`835fcec60`](https://github.com/npm/npm/commit/835fcec601204971083aa3a281c3a9da6061a7c2)
+  [#17060](https://github.com/npm/npm/pull/17060)
+  Make git repos with prepare scripts always install with both dev and prod
+  flags.
+  ([@intellix](https://github.com/intellix))
+* [`f1dc8a175`](https://github.com/npm/npm/commit/f1dc8a175eed56f1ed23bd5773e5e10beaf6cb31)
+  [#16879](https://github.com/npm/npm/pull/16879)
+  Fix support for `always-auth` and `_auth`. They are now both available in both
+  unscoped and registry-scoped configurations.
+  ([@jozemlakar](https://github.com/jozemlakar))
+* [`ddd8a1ca2`](https://github.com/npm/npm/commit/ddd8a1ca2fa3377199af74ede9d0c1a406d19793)
+  Serialize package specs to prevent `[object Object]` showing up in logs during
+  extraction.
+  ([@zkat](https://github.com/zkat))
+* [`99ef3b52c`](https://github.com/npm/npm/commit/99ef3b52caa7507e87a4257e622f8964b1c1f5f3)
+  [#17505](https://github.com/npm/npm/pull/17505)
+  Stop trying to commit updated `npm-shrinkwrap.json` and `package-lock.json` if
+  they're `.gitignore`d.
+  ([@zkat](https://github.com/zkat))
+* [`58be2ec59`](https://github.com/npm/npm/commit/58be2ec596dfb0353ad2570e6750e408339f1478)
+  Make sure uid and gid are getting correctly set even when they're `0`. This
+  should fix some Docker-related issues with bad permissions/broken ownership.
+  ([@rgrove](https://github.com/rgrove))
+  ([@zkat](https://github.com/zkat))
+* [`9d1e3b6fa`](https://github.com/npm/npm/commit/9d1e3b6fa01bb563d76018ee153259d9507658cf)
+  [#17506](https://github.com/npm/npm/pull/17506)
+  Skip writing package.json and locks if on-disk version is identical to the new
+  one.
+  ([@zkat](https://github.com/zkat))
+* [`3fc6477a8`](https://github.com/npm/npm/commit/3fc6477a89773786e6c43ef43a23e5cdc662ff8e)
+  [#17592](https://github.com/npm/npm/pull/17592)
+  Fix an issue where `npm install -g .` on a package with no `name` field would
+  cause the entire global `node_modules` directory to be replaced with a symlink
+  to `$CWD`. lol.
+  ([@iarna](https://github.com/iarna))
+* [`06ba0a14a`](https://github.com/npm/npm/commit/06ba0a14a6c1c8cdcc8c062b68c8c63041b0cec0)
+  [#17591](https://github.com/npm/npm/pull/17591)
+  Fix spurious removal reporting: if you tried to remove something that didn't
```

Comment:
It would report one removed item for everything on the command line, plus the number of files on disk it removed. So if you were removing a module w/ no transitive deps it would claim it had removed 2 modules.

Eh, what's there is fine. It's unlikely that anyone cares that much about the details. =D